sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_feature_set/feature_set_backfill_schema.py

```diff
@@ -0,0 +1,21 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from marshmallow import fields
+from azure.ai.ml._schema.core.fields import NestedField
+from azure.ai.ml._schema.core.schema import YamlFileSchema
+from azure.ai.ml._schema._feature_set.materialization_settings_schema import (
+    MaterializationComputeResourceSchema,
+)
+from azure.ai.ml._schema._feature_set.feature_window_schema import FeatureWindowSchema
+
+
+class FeatureSetBackfillSchema(YamlFileSchema):
+    name = fields.Str()
+    version = fields.Str()
+    feature_window = NestedField(FeatureWindowSchema)
+    description = fields.Str()
```

Comment:
Are description/tags part of the backfill schema? @shailshah

Reply:
here is schema definition: https://github.com/Azure/azureml_run_specification/blob/468e32ac456bcdf079d99631f150704a5a6dc9e0/schemas/FeatureSetBackfill.yaml#L4

Reply:
Yeah. Backend supports it too.